### PR TITLE
added some fixes

### PR DESCRIPTION
**Pull request by BereketRetta**
Add some fixes and files

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -75,7 +72,6 @@ const crudTodos = () => {
     }
   });
 
-  /*  EVENT LISTENERS delete completed todos */
   const removeCompleted = document.getElementById('remove-completed');
   removeCompleted.addEventListener('click', () => {
     removeComleted();
```
Comment (MrBrN197) [inline src/index.js:77]: - [ ] you have the required test but please ensure that they are passing. In the case of your tests, it looks like you have to mock the `localStorage` please make this change. the point of testing is to ensure that your code functions as expected so you need to ensure your tests are always passing.   
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @BereketRetta  👋  

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BereketRetta** [comment]
Screen Shot 2022-01-28 at 6 50 08 AM

It's passing all the required tests

**Review by rustam-tolipov — APPROVED** [review]
## Hi @BereketRetta 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**